pontoon/base/tests/managers/test_user.py

```diff
@@ -325,22 +323,3 @@ def test_mgr_user_query_args_filtering(
     assert top_contribs[0].translations_approved_count == 11
     assert top_contribs[0].translations_rejected_count == 0
     assert top_contribs[0].translations_unapproved_count == 3
-
-
-@pytest.mark.django_db
-def test_mgr_system_user_contributors_excluded(
```

Comment:
Why is this test being dropped, and why are no tests added for the changes made here?

Reply:
This test is no longer relevant, because we no longer exclude system users in that method.

But good point, I should add some test.